### DISCUSSION TITLE
Android app: folder selector

### DISCUSSION THREAD

**Discussion by thousandsofthem** [ente-io/ente#639] [Enhancements]
At this moment folder selector dialog is very much useless, it's really hard to select desired folder, example:

![Screenshot_2024-03-03-13-19-37-894_io ente photos](https://github.com/ente-io/ente/assets/736279/88692a73-14f8-4bc9-b389-984125e1925e)

It doesn't show full path either, so folders with the same name can not be easily distinguished.
Also, on my device list of folders is littered with music covers, hiding anything useful among these one-pic-folders

Proposal:
- add actual folder selector (native android?)
- show full path on photo details, album details pages
- add ability to hide folders from anywhere in the app. Simple Gallery app does it right:
![Screenshot_2024-03-03-13-27-45-847_com simplemobiletools gallery pro-edit](https://github.com/ente-io/ente/assets/736279/bbcc6668-554d-47d6-bc8e-c0ea77f85281)

**Comment by R3velation** [comment]
You don't know how much you take for granted being able to use the android file selection permission until you don't have it.  Especially when there's 700+ directories with images in them and you only want to back up one of them.

**Comment by ghost** [comment]
I second this for the same reason. My music folders contain many covers and leaflets. The only picture folder I actually want to backup is the Camera folder. For exanple, Musicolet implements this nicely and allows you to pick one or more folders to be scanned (recursively obviously). This prevents having arbitrary audio files and recordings in your music library. I wish Ente would offer the equivalent with respect to photos resp. picture files.
